article/app/controllers/ArticleController.scala

```diff
@@ -104,26 +110,35 @@ object ArticleController extends Controller with RendersItemResponse with Loggin
       renderFormat(htmlResponse, jsonResponse, article, Switches.all)
   }
 
-  def renderArticle(path: String, lastUpdate: Option[String], rendered: Option[Boolean]) = {
+  def renderArticle(path: String, lastUpdate: Option[String], rendered: Option[Boolean], page: Option[String]) = {
```

Comment:
it is then because that's how play passes it in, then later we can parse to an int.  perhaps there is a way to ask play to do the conversion like it does with Boolean?